local-volume/helm/provisioner/values.yaml

```diff
@@ -75,7 +75,7 @@ daemonset:
   #
   # Defines Provisioner's image name including container registry.
   #
-  image: quay.io/external_storage/local-volume-provisioner:latest
+  image: quay.io/external_storage/local-volume-provisioner:v2.1.0
```

Comment:
Shouldn't the default be the latest?

Reply:
The examples already contain v2.1.0, but this file wasn't updated.

I personally, like specifying a version.

Reply:
I guess my concern is that one has to remember to find and replace this every time we bump up the version. Just another thing to remember. With examples, its perhaps ok to be a bit outdated, but this config generator should always be in-synch with the latest binary from the repo (with users explicitly choosing to diverge through custom-values.yaml)

Reply:
I'm split.  I agree it's a pain to update version everytime we bump, but I also want to discourage the use of "latest" in case there are ever any backwards compatibility issues.

Reply:
Perhaps we should modify helm generation to allow 
 helm template ./helm/provisioner --set version=v2.1.0 ....

We already make the choice of provisioner version front and center in our docs, making the documentation for helm invocation also reflect that would be natural to the reader.

Reply:
what about keeping master with `latest` tag but versioned branch with version this branch is supposed to be used with? Just an idea.

Reply:
We don't have any branches in this repo, just version tags.  So when you go click on a tag and look at the example spec, it will still show "latest".

Reply:
My preference is to specify version tags, and avoid `latest` which may not have the same configuration or behavior over time.

I've created this issue to deal with updating the version information as wee go forward... https://github.com/kubernetes-incubator/external-storage/issues/701